projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -126,10 +128,17 @@ public function add_upgrades_menu() {
 		}
 
 		if ( ! $menu_exists ) {
-			add_menu_page( __( 'Upgrades', 'jetpack' ), __( 'Upgrades', 'jetpack' ), 'manage_options', 'paid-upgrades.php', null, 'dashicons-cart', 4 );
+			$site_upgrades = '%1$s<span class="inline-text">%2$s</span>';
+			$site_upgrades = sprintf(
```

Comment:
We should check if $plan is not null, otherwise the title would be something like `Updates <span class="inline-text"></span>` (e.g. on Jetpack sites).

Reply:
I was worried if this minor html pollution deserves extra code. I guess it's better checking for posterity. I also added an inline style so that the badge is not displayed if nav-unification CSS is missing. e5b9db6